plugins/infoblox/icon_infoblox/actions/add_fixed_address/action.py

```diff
@@ -1,22 +1,21 @@
-import komand
-from .schema import AddFixedAddressInput, AddFixedAddressOutput
+import insightconnect_plugin_runtime
+from .schema import AddFixedAddressInput, AddFixedAddressOutput, Input, Output, Component
 
 # Custom imports below
 
 
-class AddFixedAddress(komand.Action):
+class AddFixedAddress(insightconnect_plugin_runtime.Action):
     def __init__(self):
         super(self.__class__, self).__init__(
             name="add_fixed_address",
-            description="Add Fixed Address",
+            description=Component.DESCRIPTION,
             input=AddFixedAddressInput(),
             output=AddFixedAddressOutput(),
         )
 
     def run(self, params={}):
-        address_data = params.get("address")
+        address_data = params.get(Input.ADDRESS)
+
         ref = self.connection.infoblox_connection.add_fixed_address(address_data)
-        return {"_ref": ref}
 
-    def test(self):
-        return {"_ref": ("fixedaddress/ZG5zLmZpeGVkX2FkZHJlc3MkMTAuMTAuMTAuOC4wLi4:" "10.10.10.8/default")}
+        return {Output.REF: ref}
```

Comment:
This could be connected i.e.
```suggestion
    return {Output.REF: self.connection.infoblox_connection.add_fixed_address(address_data)}
```